rita_bin/Cargo.toml

```diff
@@ -52,6 +52,7 @@ jemallocator = {version = "0.3", optional = true}
 openssl = {version = "0.10", features = ["vendored"], optional = true}
 
 [features]
+default = ["bundle_openssl"]
```

Comment:
this looks to have snuck in from your testing and should be removed. It will bloat the size of Rita on all of our routers by move than 1mb which is quite a bit considering some devices have only 8mb of storage total, which we have to share with the operating system.

Reply:
fixed